File: mcached/java/delete/mcached_delete.java
// --------------------------------------------------------------
//	delete/mcached_delete.java
//
//					Feb/04/2015
// --------------------------------------------------------------
public class mcached_delete
{
 
// --------------------------------------------------------------
public static void main (String[] args) throws Exception
{
	System.out.println ("*** 開始 ***");

	String	key = args[0];
	System.out.println ("\tkey = " + key);

	final String server = "localhost";
	final int port = 11211;

	mcached_socket.mcached_delete_proc (server,port,key);

	System.out.println ("*** 終了 ***");
}
 
// --------------------------------------------------------------
}
// --------------------------------------------------------------

File: mcached/node/delete/mcached_delete.js
#! /usr/bin/nodejs
// -------------------------------------------------------------
//	mcached_delete.js
//
//					Feb/11/2016
// -------------------------------------------------------------
var memcache = require('memcache')

// -------------------------------------------------------------
var onConnect = function()
{
	console.log("*** 開始 ***")

	client.delete (key_in,function(error, result){ })

	client.close()

	console.log("*** 終了 ***")
}


// -------------------------------------------------------------
var key_in=process.argv[2]

console.log (key_in)

var client = new memcache.Client (11211,'localhost')
client.connect()
client.addHandler(onConnect)

// -------------------------------------------------------------

